ompi/group/group_init.c

```diff
@@ -121,12 +121,16 @@ ompi_group_t *ompi_group_allocate_plist_w_procs (ompi_proc_t **procs, int group_
     new_group->grp_my_rank = MPI_UNDEFINED;
     OMPI_GROUP_SET_DENSE(new_group);
 
+    if (NULL != orig_group) {
+        new_group->grp_instance = orig_group->grp_instance;
```

Comment:
I don't recall if OBJ_NEW will zero out the pointer - should this be initialized to NULL if not set here?

Reply:
probably, good catch